css/main.css

```diff
@@ -30,10 +30,7 @@
   background:white;
   grid-area: settings;
   padding:10px;
-  /*
-  display: none;
-  visibility: hidden;
-  */
+  overflow:hidden;
```

Comment:
Comment: There is no space between the colon and 'hidden', maybe we want to set up a linter?

Reply:
i think that has been fixed now? can you see it @khutten